EKF/covariance.cpp

```diff
@@ -641,10 +641,47 @@ void Ekf::predictCovariance()
 
 	// Don't do covariance prediction on wind states unless we are using them
 	if (_control_status.flags.wind) {
```

Comment:
@priseborough This is temporary for testing, no idea if an initial estimate of wind states based on an airspeed measurement makes sense. We could improve this by not assuming zero side slip.